### PR TITLE
Wizard recipe: xxHash_jll-v0.7.4

### PR DESCRIPTION
**Pull request by hros**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: xxHash_jll
* Version: v0.7.4

@staticfloat please review and merge.

### CODE REVIEW

**Comment by giordano** [comment]
I'm confused by this PR: `xxHash` already exists #595, how come this is creating a new file? The path looks the same to me and the parent of your commit is the latest one in this repository

File: X/xxHash_jll/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    GitSource("https://github.com/Cyan4973/xxHash.git", "173e50be0509c6fb6c1bb74d95049ef61d7fdced")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd xxHash/
+CPPFLAGS=-DXXH_INLINE_ALL make
+if [ ! -d $libdir ]; then
+  mkdir -p $libdir;
```
Comment (giordano) [inline X/xxHash_jll/build_tarballs.jl:19]: For the record, the `-p` flag to `mkdir` causes it to not throw an error if the directory already exists, so there is no need to put it under a conditional like this

File: X/xxHash_jll/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "xxHash_jll"
```
Comment (giordano) [inline X/xxHash_jll/build_tarballs.jl:5]: Oh, I see the problem: you appended `_jll` here, that's why the wizard created a new file

**Comment by hros** [comment]
Hi Mosè
The package is a "jll package" wrapping the libxxhash version 0.7.4
a previous "jll package" exists for version 0.73
the xxhash package has a julia interface for the library, and I plan to update it to use the latest "jll"

**Comment by giordano** [comment]
Sure, but you have to edit [`X/xxHash/build_tarballs.jl)`](https://github.com/JuliaPackaging/Yggdrasil/blob/6f995e9c8500825af9fe3ed9a121253349babba9/X/xxHash/build_tarballs.jl) and update the version number and the hash of the tarball, as explained in the section ["Updating the version of an existing builder"](https://github.com/JuliaPackaging/Yggdrasil/blob/6f995e9c8500825af9fe3ed9a121253349babba9/README.md#updating-the-version-of-an-existing-builder) of the README, not to create a new `build_tarballs.jl` with a different name

**Pull request closed without merging**